app/scripts/modules/core/authentication/userMenu/userRolesModal.controller.ts

```diff
@@ -0,0 +1,18 @@
+import {module} from 'angular';
+import {IModalServiceInstance} from 'angular-ui-bootstrap';
+
+class UserRolesModalCtrl {
+
+  static get $inject() { return ['$uibModalInstance', 'userRoles']; }
+
+  constructor ( private $uibModalInstance: IModalServiceInstance,
```

Comment:
Not a big deal, but we generally omit the whitespace you have here (i.e., should be `constructor(private: ...)`